metals/src/main/scala/scala/meta/internal/metals/Messages.scala

```diff
@@ -253,6 +253,12 @@ object Messages {
     def multipleMisconfiguredProjects(count: Int): String =
       s"Code navigation will not work for $count build targets in this workspace due to mis-configuration. " + moreInfo
 
+    def multipleProblemsDetected: String =
+      s"Multiple problems detected in your build."
+
+    def bazelNavigation: String =
```

Comment:
Could we suggest the flag to use to enable it?

Reply:
Currently there is no easy way to enable this, it would require changes in both metals and the users project. I think its best to work on it in a follow up.